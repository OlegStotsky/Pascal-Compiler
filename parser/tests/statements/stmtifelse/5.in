var
	a : integer;
	b : ineger;
	q : integer;

begin
	if a < b then
		if q < b then
			a := 5;
end;