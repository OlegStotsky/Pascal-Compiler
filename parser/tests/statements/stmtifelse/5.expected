Error at line 3, column 7 : couldn't find symbol ineger
